fmn/sse/server.py

```diff
@@ -173,11 +175,18 @@ def write_requests(rabbit_queue):
             subscription['queue'] = rabbit_queue
 
             while True:
-                channel, method, properties, body = yield rabbit_queue.queue.get()
-                for request in subscription['requests']:
-                    msg = u'data: ' + body.decode('utf-8') + u'\r\n\r\n'
-                    request.write(msg.encode('utf-8'))
-                yield channel.basic_ack(delivery_tag=method.delivery_tag)
+                try:
+                    channel, method, properties, body = yield rabbit_queue.queue.get()
+                    for request in subscription['requests']:
+                        msg = u'data: ' + body.decode('utf-8') + u'\r\n\r\n'
+                        request.write(msg.encode('utf-8'))
+                    yield channel.basic_ack(delivery_tag=method.delivery_tag)
+                except pika_exceptions.ConsumerCancelled:
+                    _log.info(_('The server has cancelled the AMQP consumer for '
```

Comment:
I had always assumed most logs were translated, but I confess I've never looked to see if that's a thing. If it's not I'm happy to remove it.